torch/csrc/jit/codegen/cuda/index_compute.cpp

```diff
@@ -914,7 +930,10 @@ std::unordered_map<kir::ForLoop*, kir::Val*> indexMapFromTV(
     } else if (
         (loop->iter_domain()->isBlockDim() && is_shared) ||
         (loop->iter_domain()->isThread() && is_local) ||
-        (loop->iter_domain()->parallelType() == ParallelType::Vectorize)) {
+        (loop->iter_domain()->parallelType() == ParallelType::Vectorize) ||
+        (loop->iter_domain()->parallelType() ==
+             ParallelType::MisalignedVectorize &&
+         loop->vectorize())) {
```

Comment:
Why do we need to check all three of them? Isn't `loop->vectorize()` enough?

Reply:
Yes